output/components.eslint.txt

```diff
@@ -9,6 +9,9 @@ The react/require-extension rule is deprecated. Please use the import/extensions
 /home/travis/build/Talend/ui/packages/components/src/HeaderBar/HeaderBar.component.js
   129:37  error  Elements with ARIA roles must use a valid, non-abstract ARIA role  jsx-a11y/aria-role
 
+/home/travis/build/Talend/ui/packages/components/src/IconsProvider/IconsProvider.component.js
```

Comment:
bad

Reply:
yep I don t have the choice here :/ do you want me to ignore the lint ? or we keep it so we know there is a hack.
It will be removed once all project has move to the new bundle

Reply:
ok, if we are sure that will be remove we can keep it for the moment 👍